ols/app/models/config.py

```diff
@@ -828,47 +830,81 @@ def __init__(self, **data: Any) -> None:
 class ReferenceContent(BaseModel):
     """Reference content configuration."""
 
+    vector_store_type: Optional[str] = None
     product_docs_index_path: Optional[FilePath] = None
     product_docs_index_id: Optional[str] = None
     embeddings_model_path: Optional[FilePath] = None
+    postgres: Optional[PostgresConfig] = None
 
     def __init__(self, data: Optional[dict] = None) -> None:
         """Initialize configuration and perform basic validation."""
         super().__init__()
         if data is None:
             return
 
+        self.vector_store_type = data.get(
+            "vector_store_type", constants.VectorStoreType.FAISS
+        )
+        valid_vector_store_types = list(constants.VectorStoreType)
```

Comment:
Sadly, general check on presence was only introduced in python3.12 https://docs.python.org/3/library/enum.html#enum.EnumType.__contains__

Reply:
That's sad...

Reply:
what's problem there? Conversion to list?